learn/advanced/asynchronous_updates.md

```diff
@@ -86,9 +86,9 @@ Failing to upload document:
 }
 ```
 
-## Killing MeiliSearch While a Task is Processing
+## Terminate MeiliSearch While a Task is Processing
 
-Since in MeiliSearch asynchronous tasks are <clientGlossary word="atomic"/>, killing MeiliSearch in the middle of a process does not corrupt or alter the database.
+In the rare cases where you would need to terminate MeiliSearch, since the asynchronous tasks are <clientGlossary word="atomic"/>, terminating MeiliSearch in the middle of a process does not corrupt or alter the database.
```

Comment:
```suggestion
In the rare cases when you need to terminate MeiliSearch in the middle of a process, know that it will not corrupt or alter the database in any way. This is because MeiliSearch's asynchronous tasks are <clientGlossary word="atomic"/>.
```